<ui version="4.0" >
 <class>CMakeBuildDirCreator</class>
 <widget class="QDialog" name="CMakeBuildDirCreator" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Build Directory Creator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="CMakeBuildDirChooser" native="1" name="builddirChooser" />
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="run" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPlainTextEdit" name="cmakeOutput" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="status" >
     <property name="text" >
      <string comment="KDE::DoNotExtract" >TextLabel</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMakeBuildDirChooser</class>
   <extends>QWidget</extends>
   <header>cmakebuilddirchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CMakeBuildDirCreator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>266</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CMakeBuildDirCreator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>334</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
